README.md

```diff
@@ -15,12 +15,16 @@ Kubeapps is a web-based UI for deploying and managing applications in Kubernetes
 
 Use the Helm chart to install the latest version of Kubeapps: 
 
+***Note:*** Kubeapps supports both Helm v2 and v3. For detailed instructions on how to install and use Kubeapps follow the [Getting Started Guide](docs/user/getting-started.md).
```

Comment:
```suggestion
***Note:*** Kubeapps supports both Helm 2 and Helm 3. For detailed instructions on how to install and use Kubeapps follow the [Getting Started Guide](docs/user/getting-started.md).
```
This [seems to be the official style](https://helm.sh/blog/helm-3-released/). Hence/also, I think it would improve `Ctrl+F`-ability?  WDYT?